tokio-threadpool/src/thread_pool.rs

```diff
@@ -3,7 +3,26 @@ use pool::Pool;
 use sender::Sender;
 use shutdown::Shutdown;
 
-use futures::Future;
+use futures::{Future, Poll};
+use futures::sync::oneshot;
+
+/// Handle returned from ThreadPool::spawn_handle.
+/// 
+/// This handle is a future representing the completion of a different future 
+/// spawned on to the thread pool. Created through the ThreadPool::spawn_handle 
+/// function this handle will resolve when the future provided resolves on the 
+/// thread pool.
+#[derive(Debug)]
+pub struct SpawnHandle<T, E>(oneshot::SpawnHandle<T, E>);
```

Comment:
nit: would you mind moving this to the bottom of the file? I like to try to keep the "primary" types at the top and supporting types under.

Reply:
Done.